net/nimble/host/services/bas/syscfg.yml

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+syscfg.defs:
+    BLE_SVC_BAS_BATTERY_LEVEL_PERM:
+        description: >
+            Defines permissions for writing "Battery Level" characteristics. Can
+            be zero to allow write without extra permissions or combination of:
```

Comment:
Defines permissions for *reading "Battery Level"
and
Can be zero to allow *read without extra permissions or combination of

And maybe BLE_SVC_BAS_BATTERY_LEVEL_PERM should be BLE_SVC_BAS_BATTERY_LEVEL_READ_PERM but I dont particularly care either way.

Reply:
+1 for READ_PERM